suite-common/graph/src/balanceHistoryUtils.ts

```diff
@@ -11,11 +11,17 @@ https://github.com/trezor/blockbook/blob/0afcb3a002e9f818907f2e1398f5c2af72c9f1a
 
 type BlockTime = number;
 
+type GetAccountHistoryMovementItemParams = {
+    transactions: WalletAccountTransaction[];
+    from?: number;
```

Comment:
nit(feel free to discard):

```suggestion
since?: number;
until?: number
```

Reply:
I agree this is better probably, but want to follow same API as blockbook